### DISCUSSION TITLE
Updating app to 2.0.12

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8207] [Help] [answered]
Hey,

all of my screens go empty when updating from 1.3.21 to 2.0.12.
What can I do?

**Comment by melohagan** [comment, marked as answer]
Hey @kasgithub8 

Do you see any errors in the console/network tabs (Fn+F12 on Chrome). Could you provide some screenshots - is it in the design section or the data section for example?

Do you have any layouts that need detached?

**Reply by kasgithub8** [reply]
Hi @melohagan,

the screens and data are still there. The screens just go empty like creating a new screen. It could be the layouts because we have had this app for over 6 months now and we still need to detach them. But if I recall correctly, detaching the layouts also emptied out our screens.
Here's the log: 
[budibase.log](https://github.com/Budibase/budibase/files/9752796/budibase.log)

**Reply by melohagan** [reply]
Those logs are only mentioning a Chrome extension: 

> at chrome-extension://mfidniedemcgceagapgdekdbmanojomk/content.js:1:123296

**Reply by kasgithub8** [reply]
That's the only thing in there.

**Reply by kasgithub8** [reply]
@melohagan I was able to detach the layouts without any problems this time!
Thank you for the hint.
